coordinator/Makefile

```
@@ -2,29 +2,30 @@
 
-../common/libzkp/lib/libzkp.so:
-	$(MAKE) -C ../common/libzkp/impl build
+$(LIBZKP_PATH):
+	$(MAKE) -C ./internal/logic/libzkp build
 
 clean_libzkp:
-	$(MAKE) -C ../common/libzkp/impl clean
+	$(MAKE) -C ./internal/logic/libzkp clean
 
-libzkp: clean_libzkp ../common/libzkp/lib/libzkp.so
+libzkp: clean_libzkp $(LIBZKP_PATH)
```

checkmake (0.2.2)

[warning] Target "libzkp" should be declared PHONY.

(phonydeclared)